specutils/io/default_loaders/wcs_fits.py

```diff
@@ -67,10 +76,17 @@ def wcs1d_fits_loader(file_obj, spectral_axis_unit=None, flux_unit=None,
         Units of the flux for this spectrum. If not given (or None), the unit
         will be inferred from the BUNIT keyword in the header. Note that this
         unit will attempt to convert from BUNIT if BUNIT is present.
-    hdu : int
-        The index of the HDU to load into this spectrum.
-    verbose : bool
+    hdu : int, str or None, optional
+        The index or name of the HDU to load into this spectrum
+        (default: find 1st applicable `ImageHDU`).
```

Comment:
This confuses me. Is this loader only for *image*? A spectrum can also come as table HDU, right?

Reply:
wcs1d handles only spectra with flux represented as image and spectral axis defined by the WCS; tables are covered by `tabular-fits`. That already supports masks and uncertainties, more or less, which are simply additional table columns in that representation.